CHANGELOG.md

```diff
@@ -1,9 +1,15 @@
-# [0.10.0-next.5](https://github.com/rdkcentral/firebolt-core-sdk/compare/v0.10.0-next.4...v0.10.0-next.5) (2023-02-01)
+# [1.0.0-next-major.1](https://github.com/rdkcentral/firebolt-core-sdk/compare/v0.10.0-next-major.1...v1.0.0-next-major.1) (2022-12-06)
```

Comment:
we should remove this right?
We should probably just remove anything changes for CHANGELOG and let semantic release generate it